README.md

```diff
@@ -1 +1,134 @@
-# meta-xt-prod-aos-rcar
\ No newline at end of file
+# meta-xt-prod-aos-rcar #
+
+This repository contains Renesas RCAR Gen3-specific Yocto layers for
+Xen Troops distro and `moulin` project file to build it. Layers in this
+repository provide final recipes to build meta-xt-prod-aos-rcar
+distro. This distro is the Aos product for Renesas boards.
+
+Those layers *may* be added and used manually, but they were written
+with [Moulin](https://moulin.readthedocs.io/en/latest/) build system,
+as Moulin-based project files provide correct entries in local.conf
+
+# Moulin project file
+
+Work is still in progress, but right now the following features are tested and working:
+
+* Renesas Salvator-XS M3 with 8GB memory is supported
```

Comment:
I think, just AosBox is supported

Reply:
Agreed with @klogg to keep other boards just for case.

Reply:
no problem to keep them, but READme should have the relevant information

Reply:
done